x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -179,35 +220,70 @@ async function main() {
       type: 'boolean',
       default: false,
     },
+    newUser: {
+      alias: 'nu',
+      describe: 'Adds a new user with username:password',
+      type: 'string',
+      default: 'endpoint_user:changeme',
```

Comment:
should this option (`--newUser=`) be set/enabled by default? Most params in this script are all required to be set on CLI, so maybe this one should be explicit as well sepcially since it asks for a user/password combination?

Reply:
done 5ce10629642bcdf4d08368f8b1f53374601d1046.